### PR TITLE
Layout codestyle improvements

### PR DESCRIPTION
**Pull request by n9iels**
This PR improve the codestyle of the files in the /layouts folder.
I followed the Joomla! coding standards, and made some additions that seems logic for myself to make the code better to read.

### CODE REVIEW

File: layouts/joomla/content/categories_default.php [hunk 1]
```diff
@@ -18,16 +18,16 @@
 
 <?php if ($displayData->params->get('show_base_description')) : ?>
 	<?php //If there is a description in the menu parameters use that; ?>
-		<?php if($displayData->params->get('categories_description')) : ?>
+	<?php if($displayData->params->get('categories_description')) : ?>
+		<div class="category-desc base-desc">
+		<?php echo JHtml::_('content.prepare', $displayData->params->get('categories_description'), '',  $displayData->get('extension') . '.categories'); ?>
```
Comment (zero-24) [inline layouts/joomla/content/categories_default.php]: We can add one tab here too ;)

File: layouts/joomla/content/categories_default.php [hunk 2]
```diff
@@ -18,16 +18,16 @@
 
 <?php if ($displayData->params->get('show_base_description')) : ?>
 	<?php //If there is a description in the menu parameters use that; ?>
-		<?php if($displayData->params->get('categories_description')) : ?>
+	<?php if($displayData->params->get('categories_description')) : ?>
```
Comment (zero-24) [inline layouts/joomla/content/categories_default.php]: can you add a space after the `if`?

File: layouts/joomla/content/categories_default.php [hunk 3]
```diff
@@ -18,16 +18,16 @@
...
+		<div class="category-desc base-desc">
+		<?php echo JHtml::_('content.prepare', $displayData->params->get('categories_description'), '',  $displayData->get('extension') . '.categories'); ?>
+		</div>
+	<?php else : ?>
+		<?php //Otherwise get one from the database if it exists. ?>
+		<?php  if ($displayData->parent->description) : ?>
```
Comment (zero-24) [inline layouts/joomla/content/categories_default.php]: can you remove the double space bevor the `if` :D

**Comment by n9iels** [comment]
@zero-24 thanks!

**Comment by zero-24** [comment]
One PR ;) see: https://github.com/n9iels/joomla-cms/pull/1

File: layouts/joomla/content/categories_default.php [hunk 4]
```diff
@@ -18,16 +18,16 @@
...
 <?php if ($displayData->params->get('show_base_description')) : ?>
 	<?php //If there is a description in the menu parameters use that; ?>
-		<?php if($displayData->params->get('categories_description')) : ?>
-			<div class="category-desc base-desc">
+	<?php if ($displayData->params->get('categories_description')) : ?>
+		<div class="category-desc base-desc">
 			<?php echo JHtml::_('content.prepare', $displayData->params->get('categories_description'), '',  $displayData->get('extension') . '.categories'); ?>
+		</div>
+	<?php else : ?>
+		<?php //Otherwise get one from the database if it exists. ?>
```
Comment (zero-24) [inline layouts/joomla/content/categories_default.php]: Ah missed it we need one space after the `//`

File: layouts/joomla/content/categories_default.php [hunk 5]
```diff
@@ -18,16 +18,16 @@
 
 <?php if ($displayData->params->get('show_base_description')) : ?>
 	<?php //If there is a description in the menu parameters use that; ?>
```
Comment (zero-24) [inline layouts/joomla/content/categories_default.php]: same here ;)

**Comment by zero-24** [comment]
Thanks. I have fond two more sorry ;)

**Comment by n9iels** [comment]
can someone else also toke a look at this pr so it can be merged?

**Comment by polakluk** [comment]
Hi, I tested this PR and it is working. The style changes seem consistent now. :white_check_mark:

File: layouts/joomla/content/blog_style_default_links.php [hunk 6]
```diff
@@ -13,7 +13,8 @@
 <?php foreach ($displayData->get('link_items') as $item) : ?>
 	<li>
 		<a href="<?php echo JRoute::_(ContentHelperRoute::getArticleRoute($item->slug, $item->catid, $item->language)); ?>">
-			<?php echo $item->title; ?></a>
+			<?php echo $item->title; ?>
```
Comment (andrepereiradasilva) [inline layouts/joomla/content/blog_style_default_links.php]: please don't have spaces/tabs in the anchar html tag, some browsers (ex: chrome) add an underlined space if so.

**Comment by RonakParmar** [comment]
I have tested this item :white_check_mark: successfully on 9194e23f8486a5e57fb11ae0c25e6932ac230c24<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/7382">issues.joomla.org/joomla-cms/7382</a>.</sub>

File: layouts/joomla/content/blog_style_default_links.php (hunk 6 above)
Comment (roland-d) [inline layouts/joomla/content/blog_style_default_links.php]: We should be using JHtml::_('link') here.

File: layouts/joomla/content/options_default.php [hunk 7]
```diff
@@ -11,10 +11,26 @@
 
 ?>
 <fieldset class="<?php echo !empty($displayData->formclass) ? $displayData->formclass : 'form-horizontal'; ?>">
-	<legend><?php echo $displayData->name ?></legend>
-	<?php if (!empty($displayData->description)): ?>
+	<legend><?php echo $displayData->name; ?></legend>
+	<?php if (!empty($displayData->description)) : ?>
 		<p><?php echo $displayData->description; ?></p>
 	<?php endif; ?>
+<<<<<<< HEAD
```
Comment (roland-d) [inline layouts/joomla/content/options_default.php]: @n9iels Some kind of merge conflict here, please check.

**Comment by n9iels** [comment]
@roland-d I have removed the conflict, the branch is up-to-date now :)

**Comment by RonakParmar** [comment]
I found "No newline at end of file" in "layouts/joomla/content/options_default.php" file.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/7382">issues.joomla.org/joomla-cms/7382</a>.</sub>

**Comment by n9iels** [comment]
@RonakParmar Travis says there are no problems with the codestyle

**Comment by RonakParmar** [comment]
Please check " layouts/joomla/content/options_default.php" file difference at https://github.com/joomla/joomla-cms/pull/7382/files
You can see "No newline at end of file".
7382-issue

**Comment by brianteeman** [comment]
@n9iels if you can fix that to make @RonakParmar happy I will set this RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/7382">issues.joomla.org/joomla-cms/7382</a>.</sub>

**Comment by brianteeman** [comment]
RTC doesn't need new tests for that last change<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/7382">issues.joomla.org/joomla-cms/7382</a>.</sub>

**Comment by zero-24** [comment]
@n9iels sorry but travis is still not happy.

https://travis-ci.org/joomla/joomla-cms/jobs/149221540

```
FILE: ...mla/joomla-cms/layouts/joomla/content/blog_style_default_item_title.php
--------------------------------------------------------------------------------
FOUND 1 ERROR(S) AFFECTING 1 LINE(S)
--------------------------------------------------------------------------------
 43 | ERROR | Tabs must be used to indent lines; spaces are not allowed
--------------------------------------------------------------------------------
UPGRADE TO PHP_CODESNIFFER 2.0 TO FIX ERRORS AUTOMATICALLY
--------------------------------------------------------------------------------
```

**Comment by n9iels** [comment]
@zero-24 looks like the codestyle is right this time!

**Comment by zero-24** [comment]
:+1: @n9iels

**Pull request merged**